package/desktop/setup.py

```diff
@@ -16,7 +16,7 @@
 import shutil
 
 
-VERSION = "0.30.2"
+VERSION = "0.30.3"
```

Comment:
I noticed that this was already bumped on `develop`, so shouldn't it be here too?

Reply:
it should be 0.31.0 on `develop` and `0.30.3` here, yes.